[2025-04-01T05:21:03.571+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-01T05:21:03.591+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_cron_expression_v04.task1 scheduled__2025-03-13T03:00:00+00:00 [queued]>
[2025-04-01T05:21:03.603+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_cron_expression_v04.task1 scheduled__2025-03-13T03:00:00+00:00 [queued]>
[2025-04-01T05:21:03.604+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 6
[2025-04-01T05:21:03.669+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): task1> on 2025-03-13 03:00:00+00:00
[2025-04-01T05:21:03.682+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_with_cron_expression_v04', 'task1', 'scheduled__2025-03-13T03:00:00+00:00', '--job-id', '132', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_cron_expression.py', '--cfg-path', '/tmp/tmpljdoamjp']
[2025-04-01T05:21:03.686+0000] {standard_task_runner.py:105} INFO - Job 132: Subtask task1
[2025-04-01T05:21:03.687+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=29272) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-01T05:21:03.689+0000] {standard_task_runner.py:72} INFO - Started process 29278 to run task
[2025-04-01T05:21:03.762+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_cron_expression_v04.task1 scheduled__2025-03-13T03:00:00+00:00 [running]> on host 9663692256f6
[2025-04-01T05:21:03.882+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Rajesh' AIRFLOW_CTX_DAG_ID='dag_with_cron_expression_v04' AIRFLOW_CTX_TASK_ID='task1' AIRFLOW_CTX_EXECUTION_DATE='2025-03-13T03:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-13T03:00:00+00:00'
[2025-04-01T05:21:03.884+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-01T05:21:03.906+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-04-01T05:21:03.907+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo dag with cron expression!']
[2025-04-01T05:21:03.921+0000] {subprocess.py:99} INFO - Output:
[2025-04-01T05:21:03.924+0000] {subprocess.py:106} INFO - dag with cron expression!
[2025-04-01T05:21:03.925+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-04-01T05:21:03.964+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-01T05:21:03.965+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=dag_with_cron_expression_v04, task_id=task1, run_id=scheduled__2025-03-13T03:00:00+00:00, execution_date=20250313T030000, start_date=20250401T052103, end_date=20250401T052103
[2025-04-01T05:21:04.076+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-01T05:21:04.103+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-01T05:21:04.105+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
